src/statusicon-qt/statusicon.cc

```diff
@@ -89,6 +89,9 @@ const audqt::MenuItem StatusIcon::items[] =
     audqt::MenuSep (),
     audqt::MenuCommand ({N_("_Open Files ..."), "document-open"}, StatusIcon::open_files),
     audqt::MenuCommand ({N_("Se_ttings ..."), "preferences-system"}, audqt::prefswin_show),
+#ifdef Q_OS_MACOS
+    audqt::MenuCommand ({N_("_Hide"), "hide-application"}, StatusIcon::toggle_aud_ui),
```

Comment:
Should be "Toggle" rather than "Hide", I would think ...

Reply:
No, "Hide" is the proper platform terminology, but there's actually no need for an extra platform-specific item here. There's already a "Hide Audacious" item with shortcut in the Application menu (provided by the OS).